### PR TITLE
Add ShuntCompensatorInfo

### PR DESCRIPTION
**Pull request by kgreav**
Signed-off-by: Kurt Greaves <kurt.greaves@zepben.com>

### CODE REVIEW

**Comment by ryanjcoleman** [comment]
Task linked: [PROJ-1036 Add ShuntCompensatorInfo to evolve-grpc](https://app.clickup.com/t/6929263/PROJ-1036)

File: proto/zepben/protobuf/cim/iec61968/assetinfo/ShuntCompensatorInfo.proto [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+   */
+  int32 ratedCurrent = 3;
+
+  /**
+   * Rated reactive power
+   */
+  int32 ratedReactivePower = 4;
+
+  /**
+   * Rated voltage
```
Comment (charlta) [inline proto/zepben/protobuf/cim/iec61968/assetinfo/ShuntCompensatorInfo.proto:44]: Units in dscriptions

File: proto/zepben/protobuf/np/np-requests.proto [hunk 2]
```diff
@@ -113,6 +114,11 @@ message CreateAssetInfoRequest {
     cim.iec61968.assets.AssetInfo assetInfo = 2;
 }
 
+message CreateShuntCompensatorInfoRequest {
+    int64 messageId = 1;
+    cim.iec61968.assetinfo.ShuntCompensatorInfo shuntCompensatorInfo = 2;
+}
+
```
Comment (charlta) [inline proto/zepben/protobuf/np/np-requests.proto:121]: NP has been deprecated and no new classes ahould be added

File: proto/zepben/protobuf/np/np-responses.proto [hunk 3]
```diff
@@ -50,6 +50,10 @@ message CreateAssetInfoResponse {
     int64 messageId = 1;
 }
 
+message CreateShuntCompensatorInfoResponse {
+    int64 messageId = 1;
+}
+
```
Comment (charlta) [inline proto/zepben/protobuf/np/np-responses.proto:56]: NP has been deprecated and no new classes ahould be added

File: proto/zepben/protobuf/np/np.proto [hunk 4]
```diff
@@ -42,6 +42,7 @@ service NetworkProducer {
     rpc CreatePowerTransformerInfo(CreatePowerTransformerInfoRequest) returns (CreatePowerTransformerInfoResponse);
     rpc CreateTransformerEndInfo(CreateTransformerEndInfoRequest) returns (CreateTransformerEndInfoRequest);
     rpc CreateTransformerTankInfo(CreateTransformerTankInfoRequest) returns (CreateTransformerTankInfoRequest);
+    rpc CreateShuntCompensatorInfo(CreateShuntCompensatorInfoRequest) returns (CreateShuntCompensatorInfoResponse);
```
Comment (charlta) [inline proto/zepben/protobuf/np/np.proto:45]: NP has been deprecated and no new classes ahould be added

**Review by charlta — CHANGES_REQUESTED** [review]

**Pull request merged**